packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -317,6 +328,12 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
     }
 
     async registerToolbarItems(toolbarRegistry: TabBarToolbarRegistry): Promise<void> {
+        toolbarRegistry.registerItem({
+            id: FileNavigatorCommands.REFRESH_NAVIGATOR.id,
+            command: FileNavigatorCommands.REFRESH_NAVIGATOR.id,
+            tooltip: 'Refresh',
+            priority: 0,
```

Comment:
Since we are setting the priority to 0, I believe the `Collapse All` priority should be adjusted (1) in order to be more verbose.

Reply:
Yes,the smaller the number the left-most the item will be placed 😄